project-docs/setup.md

```diff
@@ -25,3 +25,4 @@ Consider discussing your design with a classmate. Is their design different from
 
 After forking and cloning this repo you must run `yarn install` in the project directory. This will download and install all of the necessary node modules required by the project. After that finishes successfully, run `yarn start` to begin the local development server.
 
+`
```

Comment:
Ah, the old typing a random character into a file and not noticing it. Not a big deal, though.